cocos2d/core/platform/CCScreen.js

```diff
@@ -101,7 +101,11 @@ cc.screen = /** @lends cc.screen# */{
      * @returns {Boolean}
      */
     fullScreen: function () {
-        return this._supportsFullScreen && document[this._fn.fullscreenElement];
+        if(!this._supportsFullScreen)   return false;
+        else if( document[this._fn.fullscreenElement] === null)
```

Comment:
If `this._fn.fullscreenElement` not exit in document, it will equals to `undefined`